src/codewhisperer/service/transformByQHandler.ts

```diff
@@ -196,8 +196,8 @@ export async function uploadArtifactToS3(fileName: string, resp: CreateUploadUrl
             codeTransformSessionId: codeTransformTelemetryState.getSessionId(),
             codeTransformUploadId: resp.uploadId,
             codeTransformRunTimeLatency: calculateTotalLatency(apiStartTime),
-            // TODO: A nice to have would be getting the zipUploadSize
-            codeTransformTotalByteSize: 0,
+            codeTransformTotalByteSize: (await fs.promises.stat(fileName)).size,
```

Comment:
Not a blocker: This is the size of the zip folder. I believe for IntelliJ we are logging the byte size pre-zipped.

It will still help us determine overall size of upload/download

Reply:
Aligned offline, will use zip size.